radio/src/translations/cz.h.txt

```diff
@@ -833,15 +833,15 @@
 #define TR_WAITING_FOR_RX              "čekám na RX..."
 #define TR_WAITING_FOR_TX              "čekám na TX..."
 #define TR_NO_TOOLS                    "Žádné nástroje k dispozici"
-#define TR_NORMAL                      "Normal"
-#define TR_NOT_INVERTED                "Not inv"
-#define TR_NOT_CONNECTED               "!Connected"
-#define TR_CONNECTED                   "Connected"
+#define TR_NORMAL                      "Normální"
+#define TR_NOT_INVERTED                "Neinvert"
+#define TR_NOT_CONNECTED               "Nepřipojen"
+#define TR_CONNECTED                   "P5ipojen"
```

Comment:
it should be "Připojen"

Reply:
looks that your changes are not relevant to this pull-request
please check them